src/gil.rs

```diff
@@ -53,7 +53,11 @@ pub fn prepare_freethreaded_python() {
             // Signal handling depends on the notion of a 'main thread', which doesn't exist in this case.
             // Note that the 'main thread' notion in Python isn't documented properly;
             // and running Python without one is not officially supported.
+
+            // PyPy does not support the embedding API
+            #[cfg(not(PyPy))]
             ffi::Py_InitializeEx(0);
+            #[cfg(not(Py_3_7))]
```

Comment:
?

Reply:
I'm not sure but I think we have to call this function to create GIL.

Reply:
It seems that in `src/derive_utils.rs` there is this comment:
![image](https://user-images.githubusercontent.com/2467993/54073620-3dbd8600-4292-11e9-8094-a932f16e17d5.png)

Since we're initializing here I assumed this shouldn't have been here.

Reply:
I checked cpython code and it seems true, thanks.

Reply:
I think we should copy that comment

Reply:
copied.